### PR TITLE
2: Start adapting for Montreal

### PR DESCRIPTION
**Pull request by danielinteractive**

### CODE REVIEW

**Comment by danielinteractive** [comment]
@dgkf @philboileau you can also directly push your bio and photos to this branch if you like (or email still, up to you what is more convenient)

**Comment by dgkf** [comment]
@danielinteractive added an intro slide & photo for myself

File: slides/01_intro.qmd [hunk 1]
```diff
@@ -34,10 +34,13 @@ image: thumbnails/hello.jpg
 
 ::: columns
 ::: {.column width="70%"}
--   TODO
+-   Recently completed a Ph.D. in biostatistic at UC Berkeley
+-   Former intern at Genentech and Roche supervised Daniel, among others
```
Comment (danielinteractive) [inline slides/01_intro.qmd]:
```suggestion
-   Former intern at Genentech and Roche (supervised by Daniel, among others)
```
:)

File: slides/01_intro.qmd [hunk 2]
```diff
@@ -34,10 +34,13 @@ image: thumbnails/hello.jpg
...
+-   Associate at the Analysis Group in Montreal
+-   Authored and contributed to a variety of R packages available on Bioconductor, CRAN and GitHub
 -   Feel free to connect 
-    [`r fontawesome::fa("linkedin")`](https://www.linkedin.com/in/philippe-boileau-773270205/) 
-    [`r fontawesome::fa("github")`](https://github.com/philboileau)
+    [`r fontawesome::fa("globe")`](https://pboileau.ca/)
```
Comment (danielinteractive) [inline slides/01_intro.qmd]: or maybe the website in addition to LinkedIn?

Comment (danielinteractive) [reply slides/01_intro.qmd]: (up to you of course)

Comment (danielinteractive) [reply slides/01_intro.qmd]: @PhilBoileau thanks for the additions!

Comment (PhilBoileau) [reply slides/01_intro.qmd]: I'm terrible at keeping up with LinkedIn, but the more the merrier! I'll add it back. Thanks for the spot check and typo fixes by the way!

**Pull request merged**